### PR TITLE
Home page

### PR DESCRIPTION
**Pull request by domoritz**
We have to discuss what examples we want to show. I just put placeholder images in for now. 

@ZeningQu Maybe you can find some inspiration for how to write the CSS in this PR.

### CODE REVIEW

**Comment by domoritz** [comment]
Merge #1125 first

File: site/comparisons.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+---
+layout: page
+title: Vega-Lite compared with other Languages
+permalink: /comparison.html
```
Comment (kanitw) [inline site/comparisons.md]: Why is md plural, but the actual url singular?  Make it all singular?

Comment (domoritz) [inline site/comparisons.md]: Done

**Comment by domoritz** [comment]
Comments?

**Pull request merged**